pkg/controllers/crdcompatibility/objectvalidation/handle_test.go

```
@@ -1222,6 +1199,51 @@ var _ = Describe("End-to-End Admission Webhook Integration", Ordered, ContinueOn
+
+// expectCreateMatchErrorEventually is like expectInvalidCreateEventually but does not require apierrors.IsInvalid.
+func eventuallyCreate(ctx context.Context, c client.Client, obj *unstructured.Unstructured, substr string) AsyncAssertion {
+	GinkgoHelper()
+
+	return Eventually(func(g Gomega) error {
+		g.Expect(client.IgnoreNotFound(c.Delete(ctx, obj))).To(Succeed())
+		err := c.Create(ctx, obj)
+		if err == nil {
+			g.Expect(c.Delete(ctx, obj)).To(Succeed())
+		}
+		return err
```

golangci-lint (2.11.4)

[error] return with no blank line before

(nlreturn)